src/usvfs_dll/usvfs.cpp

```diff
@@ -763,6 +763,11 @@ BOOL WINAPI VirtualLinkDirectoryStatic(LPCWSTR source, LPCWSTR destination, unsi
            winapi::ex::wide::quickFindFiles(sourceP.c_str(), L"*")) {
         if (file.attributes & FILE_ATTRIBUTE_DIRECTORY) {
           if ((file.fileName != L".") && (file.fileName != L"..")) {
+
+            if (context->directoryShouldBeSkipped(file.fileName)) {
```

Comment:
Just add the negative of this to the `if` above, that would avoid an extra level and `if`.